packages/trie/README.md

```diff
@@ -95,14 +110,22 @@ Starting with the v6 release there is a new API for walking and iterating a trie
 The new walk functionality can be used like the following:
 
 ```ts
+// ./examples/trieWalking.ts
+
 import { Trie } from '@ethereumjs/trie'
+import { utf8ToBytes } from 'ethereum-cryptography/utils'
 
-const trie = await Trie.create()
-const walk = trie.walkTrieIterable(trie.root())
+async function main() {
+  const trie = await Trie.create()
+  await trie.put(utf8ToBytes('key'), utf8ToBytes('val'))
+  const walk = trie.walkTrieIterable(trie.root())
 
-for await (const { node, currentKey } of walk) {
-  // ... do something i.e. console.log( { node, currentKey } )
+  for await (const { node, currentKey } of walk) {
+    // ... do something
+    console.log({ node, currentKey })
```

Comment:
I am unable to deduce what this walk does... what does it do? It logs some nodes and key values..?

Reply:
Paging @ScottyPoi on this one.  He wrote this example (but I believe the short answer is yes)

Reply:
Yes, it seems like it's a short example of how a trie can be walked. The trie only has a single node that is logged.